functions/signin_email.js

```diff
@@ -0,0 +1,30 @@
+import { inputEmail, inputPassword } from "../tags.js";
+import { removeRedMessage, createRedMessage } from "./error_message.js";
+
+const userInfo = { "test@codeit.com": "codeit101" };
```

Comment:
유저에 대한 자료구조에 대해서 한번 생각을 해보면 좋을 듯 합니다. 현재 구조로는 key가 이메일이고 value가 비밀번호입니다. 이는 사실 email-password map 이라고 봐야할듯합니다. userInfo 라는 변수명을 보았을 때는, 단순 email-password map 구조보다는 더 많은 정보가 있을 것으로 판단됩니다. 따라서, 저는 오히려 아래와 같이 작성하는 것을 추천드립니다.

```suggestion
const registeredUsers = [{ "email": "test@codeit.com", password: "codeit101" }];
```